backend/src/mongo/models/Restaurant.js

```diff
@@ -0,0 +1,12 @@
+import mongoose from 'mongoose';
+
+export const restaurantSchema = mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  numberOfVotes: {
+    type: Number,
+    default: 0,
```

Comment:
I think we're putting numberOfVotes into the results schema. @cecilia-yin , @lilichenyuxin . What naming convention did you guys have in mind for these fields?

Reply:
Those names look fine by us.

Reply:
Yep, We are agree with this, and we are using restaurant and numberOfVotes.

Reply:
I have removed the Restaurant model, so one of you guys can implement it.